includes/Core/Assets/Assets.php

```diff
@@ -452,6 +452,7 @@ private function get_assets() {
 					'src'          => $base_url . 'js/googlesitekit-base.js',
 					'dependencies' => array(
 						'googlesitekit-base-data',
+						'googlesitekit-user-data',
```

Comment:
Why was this added? The base JS doesn't use `trackEvent` or reference the user global.

Reply:
@aaemnnosttv It's because upon page load, `window._googlesitekitUserData` is `undefined`

Reply:
I see what you mean. I wonder if these properties belong in base data instead then? 🤔 We probably should avoid loading all user data on all of wp-admin.

@felixarntz do you think we should keep all the metadata for custom dimensions in the base data or load an extra data dependency?

Reply:
@aaemnnosttv @asvinb Let's put it in `_googlesitekitBaseData`, like all other tracking dependency data. `userIDHash` for example is also technically user information, but part of the base data. That data isn't sensitive and should preferably be exlusively used for the tracking, so I think that's preferable over loading the overall user data everywhere.

Reply:
Thanks @felixarntz that sounds good to me 👍 

@asvinb please move `userRoles` to the base data as well so that tracking configuration only depends on the base data for now. In the future, we'll update it to use a separate global just for tracking data.

Reply:
Sounds good @aaemnnosttv . I've updated the PR. Can you take another look?